web/admin/components/MissionDetails/MissionDetails.js

```diff
@@ -468,6 +471,7 @@ export function MissionDetails({
                               }),
                             handleRevisionAction:
                               missionActions.editSingleActivity,
+                            allowTransfers,
```

Comment:
No need to add it in `onEditActivity`?

Reply:
From my understanding, it is not possible to edit the activity type when editing an activity (only hours for example). Therefore I don't think it's useful to know if we have transfers or not

Reply:
In fact it is useful because if you don't put it when you click on "edit activity" for a transfer activity, the label "Liaison" won't be displayed, the field will be blank.

Reply:
Ok, added